src/components/Calendar/EventColorSelector/EventColorSelector.js

```diff
@@ -12,14 +12,13 @@ import MenuItem from "@mui/material/MenuItem";
 import CircleIcon from '@mui/icons-material/Circle';
 import axios from "axios";
 import { useNavigate } from "react-router";
-import { useState } from "react";
 import PersistentDrawerLeft from '../../NavDrawer/navDrawer'
 import { CustomWhiteCard } from '../../CustomMUIComponents/CustomCards';
 
 
 export default function EventColorSelector() {
     const navigate = useNavigate();
-    const [errorMessage, setErrorMessage] = useState('')
+    const [setErrorMessage] = React.useState('')
```

Comment:
I'm not sure this implementation makes sense. If the state is never used, there is no point of having the stateSetter to begin with.

Reply:
So im not sure we are using eventColorSelector in the code. The idea was to remove the SonarCloud issues first and then understand the code to see where exactly this is implemented and if we can get rid of it

Reply:
the sonarCloud issue with this was the unused errorMessage